libs/marinade-client-rs/src/transactions/transaction_executors.rs

```diff
@@ -239,7 +251,7 @@ pub fn execute_transaction_builder(
                 &mut prepared_transaction,
                 rpc_client,
                 RpcSimulateTransactionConfig {
-                    sig_verify: !print,
+                    sig_verify: !print && !is_checked_signers,
```

Comment:
This looks like it should be the other way around - If we don't check signers, we (probably) don't want to verify signatures either, am I missing something?

Reply:
Ah. Thank you a lot. That's a mistake (coming from the prior state of the code when I named the variable `is_simulate`)